docs/src/extensions.md

```diff
@@ -85,7 +85,11 @@ also provides a `-v`/`--verbose` flag.
 
 ## `id`
 
-`id` has three additional flags: 
+`id` has three additional flags:
 * `-P` displays the id as a password file entry
 * `-p` makes the output human-readable
 * `-A` displays the process audit user ID
+
+## `uptime`
+
+`uptime` provides `-s`/`--since` to show since when the system is up.
```

Comment:
```suggestion
Similar to the proc-ps implementation and unlike GNU/Coreutils, `uptime` provides `-s`/`--since` to show since when the system is up.
```